### DISCUSSION TITLE
Disable ecarbs treatment reminder notification?

### DISCUSSION THREAD

**Discussion by nlof** [NightscoutFoundation/xDrip#2288] [Q&A]
I constantly get treatment reminders whenever I enter future ecarbs. It comes of course with sound and vibrations 😀
 Is there a way to specifically change or disable that behavior without impacting other notifications? It makes no sense since I don't need to take any action as the food is already in the gut long ago.
 I guess even being able to disable all treatment reminders would be an option for me as I don't use that feature...

Someone reported a similar (though somewhat opposite problem) previously: https://github.com/NightscoutFoundation/xDrip/issues/1162

Thank you all 

Edit: to clarify, I enter ecarbs in AAPS, those get uploaded to NightScout and the notification occurs when Xdrip imports those ecarbs treatments from Nightscout.

![Screenshot_2022-08-22-16-48-31-14](https://user-images.githubusercontent.com/70835405/185938301-a46a9e34-02aa-43b0-a35d-349acb2f87a0.jpg)
![Screenshot_2022-08-22-16-49-16-97](https://user-images.githubusercontent.com/70835405/185938311-93d2df88-05c4-4dba-8a16-42db2938e141.jpg)

**Comment by Navid200** [comment]
@psonnera

**Comment by psonnera** [comment]
Disable reminders? https://xdrip.readthedocs.io/en/latest/use/display/#enable-reminder-features but it will disable all of them...

**Comment by nlof** [comment]
Thanks, Ill give it a shot...
Had to look around, it's not in the three dot menu until you enable them in settings first. see screenshot. 
Second, I had no reminders set there, the ones I was receiving were coming from NS treatment import.
Third, it almost looks like they were already disabled by default when I opened that menu, see checked box in second screenshot... 
Not sure where else to disable them..m
I'll poke around though, I'lll set up ecarbs of 1 gram and see what happens :)

![Screenshot_2022-08-22-21-37-11-94](https://user-images.githubusercontent.com/70835405/185996326-dbbea967-35d4-4947-b9da-732f3d3241cd.jpg)
![Screenshot_2022-08-22-21-38-50-52](https://user-images.githubusercontent.com/70835405/185996330-4603dfd4-159d-4d1e-95de-e6bfc3969bb7.jpg)
![Screenshot_2022-08-22-21-38-34-22](https://user-images.githubusercontent.com/70835405/185996331-2df10c29-85fd-4a72-b2ec-98b57ad2198e.jpg)

**Comment by nlof** [comment]
Ok I wasn't able to disable them.
Alert disabled is checked off (see above) but I still get ecarb notifications (imported from NS).

There is the general notifications control for xdrip (screenshot below) but if I silence them, all notifications will be gone and apparently I receive about 60 of them daily...

Any ideas?

![Screenshot_2022-08-22-22-06-20-43](https://user-images.githubusercontent.com/70835405/186000240-ae549b44-a810-440f-870b-a94be499580b.jpg)
![Screenshot_2022-08-22-22-06-10-27](https://user-images.githubusercontent.com/70835405/186000244-b7a05534-1336-4fb0-bf8b-04978384bd7a.jpg)

**Comment by Navid200** [comment]
Do I understand correctly that the treatment is downloaded from Nightscout to xDrip and that's what's causing the notification?
Is it the notification that is shown in your logs?

**Comment by nlof** [comment]
@Navid200 Correct, it is downloaded from Nightscout to Xdrip and it's causing notification from Xdrip. 
Yes, I assume it's the one from the logs as it is at the same time. I don't have any treatment reminders set in Xdrip itself...
Only ecarbs trigger notifications, maybe because it has "delay:time" attached to it?

I edited the original post to clarify.
